Tests/TestUtils/TestBase.swift

```diff
@@ -60,7 +60,8 @@ class TestBase: XCTestCase {
         super.tearDown()
 
         // Wait .2 seconds in case there are unexpected events that were in the dispatch process during cleanup
-        usleep(200000)
+//        usleep(200000)
+        sleep(1)
```

Comment:
this change will increase the test run time with 1 sec times no of tests, what are those events that are blocking for so long before the cleanup?